cms_delete_content/README.md

```diff
@@ -0,0 +1,113 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
@pedrobaeza are you sure?
https://github.com/OCA/OCB/blob/9.0/openerp/modules/module.py#L26
https://github.com/OCA/OCB/blob/10.0/odoo/modules/module.py#L28

Once we solve this I'm gonna fix the mixing ;)